packages/browser-wallet/src/popup/pages/AddAccount/Confirm.tsx

```diff
@@ -57,7 +58,7 @@ export default function Confirm() {
             // Make request
             const expiry = Math.floor(Date.now() / 1000) + 720;
 
-            const credsOfCurrentIdentity = credentials.filter((cred) => cred.identityIndex === selectedIdentity.index);
+            const credsOfCurrentIdentity = credentials.filter(isIdentityOfCredential(selectedIdentity));
```

Comment:
So this line was the bug that Martin ran into and had Lars looking at in the mobile wallet?

Reply:
Yes, one of them.